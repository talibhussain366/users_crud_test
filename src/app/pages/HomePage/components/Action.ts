import styled from 'styled-components/macro';

export const Action = styled.button`
  font-size: 12px;
  cursor: pointer;
  margin-left: 5px;
  padding: 5px 10px;
`;
