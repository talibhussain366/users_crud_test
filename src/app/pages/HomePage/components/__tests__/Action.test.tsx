import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { Action } from '../Action';

const shallowRenderer = createRenderer();

describe('<Action />', () => {
    it('<Action />', () => {
        shallowRenderer.render(<Action />);
        const renderedOutput = shallowRenderer.getRenderOutput();
        expect(renderedOutput).toMatchSnapshot();
    });
});
