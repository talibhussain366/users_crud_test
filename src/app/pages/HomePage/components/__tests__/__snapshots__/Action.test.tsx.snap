// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Action /> <Action /> 1`] = `
<button
  className="Action-sc-167rdq9-0 iDezwI"
/>
`;
